cmake/patches/composable_kernel/Fix_Clang_Build.patch

```diff
@@ -60,21 +60,44 @@ index 514b98fde..59c8a568a 100644
 -    LDCONFIG
 -    HEADER_ONLY
 -)
+diff --git a/library/include/ck/library/tensor_operation_instance/gpu/gemm_splitk.hpp b/library/include/ck/library/tensor_operation_instance/gpu/gemm_splitk.hpp
+index 4cba0875d..07b0adb42 100644
+--- a/library/include/ck/library/tensor_operation_instance/gpu/gemm_splitk.hpp
++++ b/library/include/ck/library/tensor_operation_instance/gpu/gemm_splitk.hpp
+@@ -58,7 +58,7 @@ void add_device_gemm_xdl_splitk_f32_f32_f32_mk_nk_mn_instances(
+         DeviceGemmSplitK<Row, Col, Row, F32, F32, F32, PassThrough, PassThrough, PassThrough>>>&
+         instances);
+ #endif
+-#if(defined(CK_ENABLE_FP16) || defined(CK_ENABLE_FP8))
++#if(defined(CK_ENABLE_FP16) && defined(CK_ENABLE_FP8))
```

Comment:
These changes should be upstreamed if possible.

Reply:
I will create a fix PR to CK.